Fix build for linux python wheel

adrianlizarraga opened the pull request:
### Description
Fixes command for building Linux python packages by preventing an empty `-p` command-line option from being passed to a subsequent build script: https://github.com/microsoft/onnxruntime/blob/1f3b675453e8412e5c084bfb95997967d0c2eec2/tools/ci_build/github/linux/run_python_dockerbuild.sh#L37



### Motivation and Context
A recent [PR ](https://github.com/microsoft/onnxruntime/pull/22773)introduced a new optional command-line option (`-p`) to pass custom python exe paths. We need to check if the option is empty before forwarding the option to a separate build script.

jchen351 commented on tools/ci_build/github/linux/run_python_dockerbuild.sh:
```diff
@@ -34,7 +38,7 @@ docker run --rm \
     -e DEFAULT_TRAINING_PACKAGE_DEVICE \
     $ADDITIONAL_DOCKER_PARAMETER \
-    $DOCKER_IMAGE tools/ci_build/github/linux/build_linux_python_package.sh -c $BUILD_CONFIG -p $PYTHON_EXES $DOCKER_SCRIPT_OPTIONS
+    $DOCKER_IMAGE tools/ci_build/github/linux/build_linux_python_package.sh $DOCKER_SCRIPT_OPTIONS
```
Do we need to remove `-c $BUILD_CONFIG`?

adrianlizarraga replied:
Yes, it is already being added above in line 19. This was adding it twice. Here's a snippet of the command args in the failing pipeline:
![image](https://github.com/user-attachments/assets/e20599d8-0b63-4df7-8bb6-58d111d52c90)

jchen351 commented:
I have a separate PR to fix the same issue #22803, but I think we can merge both since both added an extra layer of protections.

adrianlizarraga commented:
> I have a separate PR to fix the same issue #22803, but I think we can merge both since both added an extra layer of protections.

@jchen351 Sounds good. Can you please take another pass at the PR? I addressed your comment.
